be/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "bugs": {
+    "url": "https://github.com/nestauk/dapsboard/issues"
+  },
+  "description": "A cache for the ElasticSearch aggregation requests performed by dapsboard",
+  "homepage": "https://github.com/nestauk/dapsboard#readme",
+  "license": "ISC",
+  "main": "index.js",
```

Comment:
we don't havve this at the moment (if we add it it should be `"main": "src/index.js",`)